src/reset.css

```diff
@@ -0,0 +1,5 @@
+body,
+div,
+p {
+  margin: 0;
```

Comment:
does div have default marging?

Reply:
I do not really understand what should be fixed

Reply:
https://github.com/mate-academy/layout_product-cards/pull/2342/files#r1057004376
my question was: does div have default marging?
answer: no, it doesn't. so why shoul we reset it?